### PR TITLE
update document for option no dereference.

### PR DESCRIPTION
**Pull request by vwxyzh**

### CODE REVIEW

File: Documentation/tutorial/docfx.exe_user_manual.md [hunk 1]
```diff
@@ -217,6 +217,7 @@ maxParallelism           | Set the max parallelism, 0 (default) is same as the c
 markdownEngineName       | Set the name of markdown engine, default is `dfm`, and another build-in engine is `gfm`.
 markdownEngineProperties | Set the parameters for markdown engine, value should be a JSON string.
 noLangKeyword            | Disable default lang keyword, it can be downloaded from [here](http://dotnet.github.io/docfx/langwordmapping/langwordMapping.yml).
+noDereference            | If set to true, manifest will keep link form, and files do *NOT* copy to output folder.
```
Comment (superyyrrzz) [inline Documentation/tutorial/docfx.exe_user_manual.md]: keep in link form
will *NOT* be copied to

**Comment by hellosnow** [comment]
:shipit:

File: Documentation/tutorial/docfx.exe_user_manual.md [hunk 2]
```diff
@@ -217,6 +217,7 @@ maxParallelism           | Set the max parallelism, 0 (default) is same as the c
 markdownEngineName       | Set the name of markdown engine, default is `dfm`, and another build-in engine is `gfm`.
 markdownEngineProperties | Set the parameters for markdown engine, value should be a JSON string.
 noLangKeyword            | Disable default lang keyword, it can be downloaded from [here](http://dotnet.github.io/docfx/langwordmapping/langwordMapping.yml).
+keepFileLink             | If set to true, docfx does not dereference (aka. copy) file to the output folder, instead, it saves a link_to_path property inside mainfiest.json to indicate the physical location of that file.
```
Comment (vicancy) [inline Documentation/tutorial/docfx.exe_user_manual.md]: also list cases when link_to_path is used? e.g. incremental build, the same file content...

File: Documentation/tutorial/docfx.exe_user_manual.md [hunk 3]
```diff
@@ -217,7 +217,7 @@ maxParallelism           | Set the max parallelism, 0 (default) is same as the c
 markdownEngineName       | Set the name of markdown engine, default is `dfm`, and another build-in engine is `gfm`.
 markdownEngineProperties | Set the parameters for markdown engine, value should be a JSON string.
 noLangKeyword            | Disable default lang keyword, it can be downloaded from [here](http://dotnet.github.io/docfx/langwordmapping/langwordMapping.yml).
-keepFileLink             | If set to true, docfx does not dereference (aka. copy) file to the output folder, instead, it saves a link_to_path property inside mainfiest.json to indicate the physical location of that file.
+keepFileLink             | If set to true, docfx does not dereference (aka. copy) file to the output folder, instead, it saves a `link_to_path` property inside `mainfiest.json` to indicate the physical location of that file. A file link will be created by incremental build and copy resouce file.
```
Comment (vicancy) [inline Documentation/tutorial/docfx.exe_user_manual.md]: typo: resouce-> resource

**Pull request merged**